File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/calls/VerifyDependencyCallViewModelTestEnvironmentImpl.cpp
#include "VerifyDependencyCallViewModelTestEnvironmentImpl.h"
#include <memory>
#include "VerifyDependencyCallViewModel.h"
#include "dependencies/LoggerSpy.h"
#include "dependencies/SimpleDependency_ViewModel.h"

namespace dependencies::calls
{
  void VerifyDependencyCallViewModelTestEnvironmentImpl::Init()
  {
  }
  std::shared_ptr<VerifyDependencyCallViewModel> VerifyDependencyCallViewModelTestEnvironmentImpl::BuildSut()
  {
    return nullptr;
  }
  std::shared_ptr<LoggerSpy> VerifyDependencyCallViewModelTestEnvironmentImpl::getLogger()
  {
    return nullptr;
  }
  std::shared_ptr<SimpleDependency_ViewModel> VerifyDependencyCallViewModelTestEnvironmentImpl::getSimpleDependency_ViewModel()
  {
    return nullptr;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/dependencies/calls/VerifyDependencyCallViewModelTestEnvironmentImpl.cs
/// <filename>
///     VerifyDependencyCallViewModelTestEnvironmentImpl.cs
/// </filename>
using dependencies.calls;
using dependencies;

namespace dependencies.calls
{
    public class VerifyDependencyCallViewModelTestEnvironmentImpl : dependencies.calls.VerifyDependencyCallViewModelTestEnvironment
    {
        public override void Init()
        {
        }

        public override dependencies.calls.VerifyDependencyCallViewModel BuildSut()
        {
            return null;
        }

        public override dependencies.LoggerSpy getLogger()
        {
            return null;
        }

        public override dependencies.SimpleDependency_ViewModelSpy getSimpleDependency_ViewModel()
        {
            return null;
        }
    }
}
